### ISSUE TITLE
[BUG] Uncertainty about GetContainerItemInfo() change to return number of charges

### ISSUE THREAD

**Issue by veechs** [balakethelock/SuperWoW#3]
**Describe the bug**
When I call `GetContainerItemInfo` on an item that has charges, such as [Manual Crowd Pummeler](https://www.wowhead.com/classic/item=9449/manual-crowd-pummeler) or [Taming Rod](https://www.wowhead.com/classic/item=15916/taming-rod), I receive `1` as the `itemCount`, even though the SuperWoW docs seem to indicate it should be the number of charges instead. Please let me know if I'm misunderstanding the expected behavior.

**To Reproduce**
1. Add [Taming Rod](https://www.wowhead.com/classic/item=15916/taming-rod) to your inventory
2. Call `GetContainerItemInfo(<bagNum>, <slotNum>)`
3. Observe that the return values are: `Interface\Icons\Ability_Hunter_BestTaming, 1, nil, -1, nil`

**Expected behavior**
I *think* it's supposed to return `Interface\Icons\Ability_Hunter_BestTaming, 3, nil, -1, nil` with SuperWoW loaded.

**Additional context**
I've verified that SuperWoW is active, since the `CombatLogAdd` function exists.

**Comment by balakethelock** [comment]
Indeed, items with charges fall into two categories, there are items that on running out of charges, the item with disappear (like tea with sugar or wizard oil) which are the ones working with this lua function at the moment. In the future I will make it also work with the 2nd category (item stays in inventory unusable after it runs out of charges)

**Comment by veechs** [comment]
Thanks for the explanation! I just tested with Wizard Oil and Tea With Sugar and they are indeed returning charge counts. It might be good to update the documentation to note that charges are returned as negative numbers.
